tools/cache_perf_tool/processEntry.go

```diff
@@ -74,6 +85,16 @@ func (p *processEntryGen) nodeWrites(ctx context.Context, node *sgwNode, delay t
 	ticker := time.NewTicker(delay)
 	defer ticker.Stop()
 	for {
+		chanMap = make(channels.ChannelMap)
+		for i := 0; i < p.numChansPerDoc; i++ {
+			if chanCountWait == p.totalChans {
+				// when count gets to total number configured channels we rest index to 0
+				chanCountWait = 0
+			}
+			chanName := "test-" + strconv.Itoa(chanCountWait)
+			chanMap[chanName] = nil
+			chanCountWait++
+		}
 		timeStamp := time.Now()
```

Comment:
Do you want timestamp to be before the select, or do you want this to be after the ticker is selected?